fleet/api/http/transport.go

```diff
@@ -104,6 +104,26 @@ func MakeHandler(tracer opentracing.Tracer, svcName string, svc fleet.Service) h
 		decodeView,
 		types.EncodeResponse,
 		opts...))
+	r.Get("/backends/agents", kithttp.NewServer(
+		kitot.TraceServer(tracer, "list_backends")(listAgentBackendsEndpoint(svc)),
+		decodeListBackends,
+		types.EncodeResponse,
+		opts...))
+	r.Get("/backends/agents/:id/handler", kithttp.NewServer(
+		kitot.TraceServer(tracer, "view_agent_backend_handler")(viewAgentBackendHandlerEndpoint(svc)),
+		decodeView,
+		types.EncodeResponse,
+		opts...))
+	r.Get("/backends/agents/:id/input", kithttp.NewServer(
```

Comment:
I believe this is where we should allow the agent backends to add their own Endpoints. That way pktvisor backend can add the input, handler, and tap endpoints itself